Hugging Face/Hugging_Face_Few_Shot_Learning_with_Inference_API.ipynb

```diff
@@ -184,35 +161,32 @@
     "### Add the Model and API token\n",
     "\n",
     "#### Steps to get API token\n",
-    "- Create an account on Hugging Face\n",
+    "- Create an account on [Hugging Face](https://huggingface.co)\n",
     "- Log in, and click on profile icon (top right corner)\n",
     "- Go to settings\n",
-    "- Click on Access tokens\n",
-    "- Now, create a new access token with name: \"gpt-inference\" and role: \"read\"\n",
+    "- Click on [Access tokens](https://huggingface.co/settings/tokens)\n",
+    "- Now, create a new access token with name: `GPT_INFERENCE` and role: `read`\n",
     "- Copy the generated token and paste it below\n",
     "\n",
     "We will use <a href=\"https://huggingface.co/EleutherAI/gpt-neo-1.3B\">gpt-neo-1.3B</a> model for our demonstration. "
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 26,
+   "execution_count": null,
    "id": "5d02938f",
    "metadata": {
-    "execution": {
-     "iopub.execute_input": "2023-11-08T19:11:34.006773Z",
-     "iopub.status.busy": "2023-11-08T19:11:34.006521Z",
-     "iopub.status.idle": "2023-11-08T19:11:34.010795Z",
-     "shell.execute_reply": "2023-11-08T19:11:34.010166Z",
-     "shell.execute_reply.started": "2023-11-08T19:11:34.006746Z"
-    },
     "papermill": {},
     "tags": []
    },
    "outputs": [],
    "source": [
+    "# Uncomment to store the environment variable in Naas cloud\n",
+    "# naas.secret.add(\"GPT_INFERENCE\", \"Paste_token_here\")\n",
+    "\n",
+    "# Set the environment variables\n",
     "MODEL = \"EleutherAI/gpt-neo-1.3B\"\n",
```

Comment:
@srsawant34 Can you justify the reason for choosing the model over others? Anything specific that makes it unique as such. Add thus in the markdown above the cell. And will be good to go.